chain.schema.json

```
@@ -3,7 +3,9 @@
   "codebase":{
     "git_repo": "https://github.com/osmosis-labs/osmosis",
     "recommended_version": "12.1.0",
-    "compatible_versions": ["12.1.0"],
+    "compatible_versions": [
+      "12.1.0"
```

```suggestion
      "12.2.0"
```

taking this as a chance to fix this